app/models/label_fields.rb

```diff
@@ -63,17 +63,17 @@ def initialize(labels)
     ##
     # find value for key
     def find(key)
-      values[key.to_sym]
+      values[key]
     end
 
     ##
-    # Labels is of type OpenStruct
+    # Labels is a hash
```

Comment:
Not sure the relevance of this comment for the method?

Reply:
I have absolutely no idea.